apps/web/package.json

```diff
@@ -71,7 +71,7 @@
     "@stripe/react-stripe-js": "^1.10.0",
     "@stripe/stripe-js": "^1.35.0",
     "@tanstack/react-query": "^5.17.15",
-    "@tremor/react": "^2.0.0",
+    "@tremor/react": "^3.18.7",
```

Comment:
- Without upgrade, we get `TypeError: Cannot read properties of undefined (reading 'ReactCurrentBatchConfig')` error when using its components
- Apparently, ReactCurrentBatchConfig is specific to React 18